src/main/java/com/example/petbuddybackend/entity/notification/Notification.java

```diff
@@ -0,0 +1,46 @@
+package com.example.petbuddybackend.entity.notification;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+
+import java.time.ZonedDateTime;
+
+@MappedSuperclass
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@SuperBuilder
+public class Notification {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false)
+    private Long objectId;
+
+    private String objectType;
+
+    @Column(nullable = false, updatable = false)
+    ZonedDateTime createdAt;
+
+    @Column(nullable = false, length = 4000)
+    private String message;
+
+    @Column(nullable = false)
+    private boolean isRead;
```

Comment:
To isRead jest teraz gdzieś aktualizowane?

Reply:
jeszcze nie ten task :)

Reply:
ale to endpointem prawodpodobnie pojdzie